software/gatk4/variantfiltration/main.nf

```diff
@@ -20,7 +20,9 @@ process GATK4_VARIANTFILTRATION {
 
     input:
     tuple val(meta), path(vcf)
-    tuple path(fasta), path(fai), path(dict)
+    path(fasta)
+    path(fai)
+    path(dict)
 
     output:
     tuple val(meta), path("*filtered.vcf"), emit: vcf
```

Comment:
```suggestion
    tuple val(meta), path("*.vcf"), emit: vcf
```

Reply:
just *.vcf won't work for the output right? As the input vcf will be in the workdir too. I can instead do: "*${options.suffix}.vcf". Which would work.